libsolidity/codegen/CompilerContext.h

```diff
@@ -56,7 +56,9 @@ class CompilerContext
 			m_runtimeSub = size_t(m_asm->newSub(m_runtimeContext->m_asm).data());
 	}
 
+	/// Update currently enabled set of experimental features.
 	void setExperimentalFeatures(std::set<ExperimentalFeature> const& _features) { m_experimentalFeatures = _features; }
+	/// @returns true if the given feature is enabled.
```

Comment:
Which of the source units does it check?

Reply:
The above supplied list.

Reply:
Ah right :-)